src/cli/onefuzz/templates/regression.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import json
+from typing import Dict, List, Optional
+
+from onefuzztypes.enums import ContainerType, TaskDebugFlag, TaskType
+from onefuzztypes.models import NotificationConfig, RegressionReport
+from onefuzztypes.primitives import Container, Directory, File, PoolName
+
+from onefuzz.api import Command
+
+from . import JobHelper
+
+
+class Regression(Command):
+    """ Regression job """
+
+    def _check_regression(self, container: Container, file: File) -> bool:
+        content = self.onefuzz.containers.files.get(Container(container), file)
+        as_str = content.decode()
+        as_obj = json.loads(as_str)
+        report = RegressionReport.parse_obj(as_obj)
+
+        if report.crash_test_result.crash_report is not None:
+            return True
+
+        if report.crash_test_result.no_repro is not None:
+            return False
+
+        raise Exception("invalid crash report")
+
+    def generic(
+        self,
+        project: str,
+        name: str,
+        build: str,
+        pool_name: PoolName,
+        *,
+        reports: Optional[List[str]] = None,
+        crashes: Optional[List[File]] = None,
+        target_exe: File = File("fuzz.exe"),
+        tags: Optional[Dict[str, str]] = None,
+        notification_config: Optional[NotificationConfig] = None,
+        target_env: Optional[Dict[str, str]] = None,
+        setup_dir: Optional[Directory] = None,
+        reboot_after_setup: bool = False,
+        target_options: Optional[List[str]] = None,
+        dryrun: bool = False,
+        duration: int = 24,
+        crash_report_timeout: Optional[int] = None,
+        debug: Optional[List[TaskDebugFlag]] = None,
+        check_retry_count: Optional[int] = None,
+        check_fuzzer_help: bool = True,
+        delete_input_container: bool = True,
+        check_regressions: bool = False,
+    ) -> None:
+        """
+        generic regression task
+
+        :param File inputs: Specify a directory of inptus to use in the regression task
```

Comment:
Typo, but also, what parameter is this referring to?

Reply:
fixed